commons-rng-sampling/src/main/java/org/apache/commons/rng/sampling/distribution/ContinuousUniformSampler.java

```diff
@@ -196,17 +196,10 @@ private static boolean validateOpenInterval(double lo, double hi) {
             // (MAX_VALUE has all bits set except the most significant sign bit.)
             bitsx &= Long.MAX_VALUE;
             bitsy &= Long.MAX_VALUE;
-            if (lessThanUnsigned(bitsx + bitsy, MIN_ULP_OPPOSITE_SIGN)) {
-                return false;
-            }
-        } else {
-            // Same signs, subtraction will count the ULP difference.
-            // This should be above 1.
-            if (Math.abs(bitsx - bitsy) < MIN_ULP_SAME_SIGN) {
-                return false;
-            }
+            return !lessThanUnsigned(bitsx + bitsy, MIN_ULP_OPPOSITE_SIGN);
         }
-        return true;
+        // maintain existing comment ...
```

Comment:
The `maintain existing comment ...` was shorthand for leaving the existing comment in place. So please put it back:
```Java
    // Same signs, subtraction will count the ULP difference.
    // This should be above 1.
```

Reply:
if the code need comment, probably need to fixed.

Changed